frontend/src/features/projects/common/projectConverter.ts

```diff
@@ -90,66 +90,75 @@ export const getFlatProjectNotes = (project: IApiProject) => {
   return notes;
 };
 
+export const toFlatProperty = (apiProperty: IApiProperty, pp?: IProjectProperty): IProperty => {
+  const assessedLand = isParcelOrSubdivision(apiProperty as any)
+    ? getCurrentYearEvaluation(apiProperty.evaluations, EvaluationKeys.Assessed)
+    : undefined;
+  const assessedBuilding = getCurrentYearEvaluation(
```

Comment:
the assessed building should either come from the improvements evaluations if the current property is a parcel, or just from the assessment evaluations if this property is a building.